lib/CachedSource.js

```diff
@@ -14,25 +14,18 @@ class CachedSource extends Source {
 		this._cachedSize = undefined;
 		this._cachedMaps = {};
 
-		if(source.node) this.node = function(options) {
-			return this._source.node(options);
-		};
+		if(source.node) this.node = options => this._source.node(options);
 
-		if(source.listMap) this.listMap = function(options) {
-			return this._source.listMap(options);
-		};
+		if(source.listMap) this.listMap = options => this._source.listMap(options);
 	}
 
 	source() {
-		if(typeof this._cachedSource !== "undefined") return this._cachedSource;
-		return this._cachedSource = this._source.source();
+		return typeof this._cachedSource !== "undefined" ? this._cachedSource : this._cachedSource = this._source.source();
```

Comment:
```js
if (typeof this._cacheSource === 'undefined') {
  this._cacheSource = this._source.source();
}

return this._cacheSource;
```

Reply:
Hi, thanks for reviewing the PR. I have refactored the above code with ternary conditionals. Could you help me understand why we should revert this change ?

As far as the logic goes, I believe it is right. I have reproduced a similar example here: https://repl.it/@abiduzz420/BusyUnripeReality

Reply:
It's only a suggestion, because of the assignment in the `return` => `: this._cachedSource = this._source.source();` here, but as always with code style debatable of course :). And be switching the conditional check a ternary isn't necessary anymore.

Reply:
Okay! That sounds fair. I shall revert it. Thanks 👍